web/src/lib/roadmap/fixtures/tasks/blocked-task.md

```diff
@@ -2,6 +2,7 @@
 id: "blocked-id"
 urlSlug: "blocked-url-slug"
 name: "Blocked Task"
+stepNumber: 1
```

Comment:
this comment applies to all the task files changed in `/fixtures/tasks` - I thought that the step number was coming from the add-ons, right? Not from the task file itself. I think these are probably just leftover unused lines of code that can be removed